metricbeat/module/mongodb/status/status_integration_test.go

```diff
@@ -33,33 +32,40 @@ import (
 func TestFetch(t *testing.T) {
 	compose.EnsureUp(t, "mongodb")
 
-	f := mbtest.NewEventFetcher(t, getConfig())
-	event, err := f.Fetch()
-	if !assert.NoError(t, err) {
+	f := mbtest.NewReportingMetricSetV2(t, getConfig())
+	events, errs := mbtest.ReportingFetchV2(f)
+
+	assert.Empty(t, errs)
+	if !assert.NotEmpty(t, events) {
 		t.FailNow()
 	}
 
-	t.Logf("%s/%s event: %+v", f.Module().Name(), f.Name(), event)
+	t.Logf("%s/%s event: %+v", f.Module().Name(), f.Name(),
+		events[0].BeatEvent("haproxy", "status").Fields.StringToPrint())
+
+	event := events[0].BeatEvent("haproxy", "status").Fields
```

Comment:
"haproxy" again? Rather "mongodb"? :-)

Reply:
That is what happens when working on 2 PR's in parallel. I was so consitent with haproxy that the tests even passed :-)